lib/model/card/action/scry_action.dart

```diff
@@ -17,4 +17,7 @@ class ScryAction extends CardAction {
   CardActionDetails get actionDetails =>
       CardActionDetails(cost, description,
           cardsToScry: numToScry);
+
+  @override
+  bool autoProgress() => false;
```

Comment:
We need this, as otherwise the state progresses to the next card before you get a chance to scry (so even if you put something on top, rather than getting it immediately you have to action the next card to get to it).